WcaOnRails/app/controllers/admin_controller.rb

```diff
@@ -75,4 +75,16 @@ def person_data
       dob: @person.dob,
     }
   end
+
+  def compute_auxiliary_data
+    @auxiliary_data_start = Timestamp.find_or_create_by(name: 'auxiliary_data_start')
+    @auxiliary_data_end = Timestamp.find_or_create_by(name: 'auxiliary_data_end')
+  end
+
+  def do_compute_auxiliary_data
+    # Reset the end date. The start date is updated directly before the computation.
+    Timestamp.find_or_create_by(name: 'auxiliary_data_end').update date: nil
+    ComputeAuxiliaryData.perform_later
```

Comment:
But this is kinda problematic because of our `if` statement, we don't want the end date to be set if we deferred the job, what do you think?